package.json

```diff
@@ -9,22 +9,22 @@
     "url": "git+https://github.com/apollostack/subscriptions-transport-ws.git"
   },
   "dependencies": {
-    "@types/ws": "0.0.38",
+    "@types/ws": "0.0.40",
     "backo2": "^1.0.2",
-    "eventemitter3": "^2.0.2",
-    "graphql": "^0.9.1",
-    "graphql-subscriptions": "^0.3.0",
-    "graphql-tag": "^1.2.4",
+    "eventemitter3": "^2.0.3",
+    "graphql": "^0.9.3",
+    "graphql-subscriptions": "^0.3.1",
+    "graphql-tag": "^2.0.0",
     "lodash.isobject": "^3.0.2",
     "lodash.isstring": "^4.0.1",
-    "ws": "^1.1.0"
+    "ws": "^2.3.1"
   },
   "scripts": {
     "compile": "tsc",
     "pretest": "npm run compile",
     "test": "npm run testonly --",
     "posttest": "npm run lint",
-    "lint": "tslint ./src/**/*.ts",
+    "lint": "tslint --type-check --project ./tsconfig.json ./src/**/*.ts",
```

Comment:
@DxCx those changes weren't made by me. Is an already merged pull request into master :(